templates/backend/interfaces/.gitignore

```
@@ -1,2 +1,4 @@
 props/
 *_gen.go
+cmd/
+mock_jsons/
```

```suggestion
cmd/
mock_jsons/

```
もう一点、cmdの中身が一度addされているようなので消してください

---

4b66e1d で対応